lib/cn/csched_sp3.c

```diff
@@ -209,7 +200,7 @@ struct sp3 {
     uint             jobs_started;
     uint             jobs_finished;
     uint             jobs_max;
-    uint             rr_job_type;
+    uint             rr_wtype;
```

Comment:
What is rr?

Reply:
Round-robin.  Probably should just be "wtype_cur", as the round-robin-ness is an artifact of how sp3_schedule() manages this variable...